DESCRIPTION

```diff
@@ -19,4 +19,4 @@ LinkingTo: Rcpp
 SystemRequirements: 
     grpc (>= 4.0.0)
     protobuf
-RoxygenNote: 6.0.1
+RoxygenNote: 7.1.0
```

Comment:
Was this really necessary?

Reply:
Well, for documentation purposes, it is necessary to run the devtools::document() command. I already did this before to document my code updates. Neal recommended me to rerun roxygen after the last update so I did, and yes I do think it is necessary!

Reply:
Is the documentation included within the commits?

Reply:
Well, no problems at all! Actually I didn't exactly get your point but it looks like you figured it out! However, if you still have a doubt about any of my commits please let me know.

Reply:
It was just a friendly review :-)

Reply:
Thanks for the review! It is my pleasure to get more of your reviews about any future commits.
I will absolutely be glad to look over your solution!